app/src/main/res/values-ru-rRU/strings.xml

```diff
@@ -74,7 +74,7 @@
     <string name="msg_delete_account">Удалить аккаунт</string>
     <string name="msg_change_status">Изменить статус</string>
 
-    <string-array name="languages"> <!-- TODO Add translations -->
+    <string-array name="языки">
```

Comment:
@Pitstopper This is making the app not building. You need to translate the strings bellow (English, Arabic, German, etc) not the name of the `string-array`.

Reply:
Fixes made